apps/files/src/components/FileDetails.vue

```diff
@@ -51,9 +55,12 @@ export default {
       this.activeTab = app
     },
     clipboardSuccessHandler () {
-      this.showMessage({
-        title: this.$gettext('The private link has been copied to your clipboard.')
-      })
+      this.linkCopied = true
+
+      // Use copy icon after some time
+      setTimeout(() => {
+        this.linkCopied = false
+      }, 1000)
```

Comment:
Maybe set it to 3 seconds? 1 feels a little bit too quick. But that's just a detail - feel free to merge without it.

Reply:
github changes the icon after one sec as well :man_shrugging: